rlog/DRAFT-fiat-shamir.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Vac 101: Transforming an Interactive Protocol to a Noninteractive Argument'
+date: 2024-07-01 12:00:00
+authors: marvin
+published: true
+slug: Vac-101-Fiat-Shaimr
+categories: research
+
+
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+
+In this post, provide a discussion on a common technique used to convert interactive protocols to their noninteractive variant.
+<!--truncate-->
+
+## Introduction
+
+The set of interactive protocols form a class of protocols that consist of communication between two parties: the Prover and the Verifier.
+The Prover tries to convince the Verifier of a given claim.
+For example, the Prover may want to convince the Verifier that she owns a specific UTXO;
+that is, the Prover possesses the ability to spend the UTXO.
+In many instances, there is information that the Prover does not wish to reveal to the Verifier.
+In our example, it is critical that the Prover does not provide the Verifier with the spending key associated to her UTXO.
+In addition to the Prover's claim and secret data, there is additional data, public parameters, that the claimed statement is expressed in terms of.
+The public parameters can be thought of as the basis for all similar claims.
+
+In an interactive protocol, the Prover and the Verifier are in active communication.
```

Comment:
Maybe we could include an example of a sigma protocol (for instance). Something basic, just to provide a picture of an interactive protocol.

Reply:
Moved examples of Schnorr and Chaum-Pedersen up, as well as provided an empty framework diagram (similar to Schnorr)